src/js/azdata/azdataQueryPlan.js

```diff
@@ -302,20 +302,44 @@ azdataQueryPlan.prototype.init = function (container, iconPaths, badgeIconPaths)
     graph.convertValueToString = function (cell) {
         if (cell.value != null && cell.value.label != null) {
             let hasWindowsEOL = cell.value.label.includes('\r\n');
+            const joinStrings = (strArray) => {
+                if (hasWindowsEOL) {
+                    return strArray.join('\r\n');
+                }
+                else {
+                    return strArray.join('\n');
+                }
+            };
+
+
             let splitLabel = cell.value.label.split(/\r\n|\n/);
-            let cellLabel = splitLabel.map(str => {
+            let cellLabel = splitLabel.map((str, index) => {
                 let label = '';
-                label += str;
+
+                if (index === 0) {
+                    label += str.replace(/\(([^)]+)\)/g, '');
+                }
+                else if (index === 1 && splitLabel.length >= 3 && str.includes('].[')) {
```

Comment:
Since we are only supporting SQL query plans, I think this is fine. However, I am not so sure how this solution will  generalize across other database servers. Maybe this logic should be present in SQLToolService. @kburtram what do you think about this?